extensions/spark/kyuubi-spark-authz/src/test/scala/org/apache/kyuubi/plugin/spark/authz/gen/DeltaCommands.scala

```diff
@@ -53,8 +54,16 @@ object DeltaCommands extends CommandSpecs[TableCommandSpec] {
     TableCommandSpec(cmd, Seq(tableDesc), queryDescs = Seq(queryDesc))
   }
 
+  val OptimizeTableCommand = {
+    val cmd = "org.apache.spark.sql.delta.commands.OptimizeTableCommand"
+    val childDesc = TableDesc("child", classOf[ResolvedTableTableExtractor])
+    val tableDesc = TableDesc("tableId", classOf[TableIdentifierOptionTableExtractor])
+    TableCommandSpec(cmd, Seq(childDesc, tableDesc), OPTIMIZETABLE)
```

Comment:
Let’s use MSCK here?

Reply:
Do you mean to remove the new OperationType of OPTIMIZETABLE and use MSCK directly? Is it better to make a difference? After all, they are two kinds of operations.

Reply:
Let's keep the enum as a subset of https://github.com/apache/hive/blob/master/ql/src/java/org/apache/hadoop/hive/ql/security/authorization/plugin/HiveOperationType.java

Reply:
How about ALTERTABLE_COMPACT?

Reply:
Nice choice！